workers/generic-worker/multiuser_windows.go

```diff
@@ -30,6 +33,7 @@ func platformFeatures() []Feature {
 		&RunTaskAsCurrentUserFeature{},
 		&RDPFeature{},
 		&RunAsAdministratorFeature{}, // depends on (must appear later in list than) OSGroups feature
+		&InteractiveFeature{},
```

Comment:
```suggestion
```

This can now be moved to the global features list across all platforms https://github.com/taskcluster/taskcluster/blob/023387cb237ff46f60a3a8838504e850c356c20d/workers/generic-worker/main.go#L74-L79.

Please be sure to remove `InteractiveFeature` from the various `platformFeatures()` functions. Looks like `workers/generic-worker/insecure_other.go` can safely be removed entirely, also.

Reply:
`vet: ./main.go:82:30: undefined: platformFeatures` can't remove the file no, have to return an empty list of features.